app/jobs/job_helpers/usps_proofing_results_job/analytics_helper.rb

```diff
@@ -0,0 +1,83 @@
+module JobHelpers
+  module UspsProofingResultsJob
+    module AnalyticsHelper
```

Comment:
@NavaTim I'm curious, why do you think that's the better approach?

Reply:
So @NavaTim I'm running into an issue where fake_analytics.rb is not aware of the analytics calls I'm making from this helper class. I'm not sure if I'm going to need to make a new Fake_Analytics class to be aware of these methods or not. If I need to refactor Fake_analytics.rb to make this change work, I might be inclined to shelve this refactor until a later time and fully separate the work. I think scope creep is starting to become an issue here for me.

Reply:
It looks like you're making the classes stateful, but it's better to keep them stateless, esp. in a process like this job where many different records are processed that differ in that type of state. I would suggest completely hiding the fact that a new `Analytics` instance may be constructed inside `AnalyticsHelper` by taking the `User` object as a parameter for each of the methods (as you had in the module version).

I expect this will require some changes to tests since all of this logic was previously encapsulated in the job. I would suggest initially going about this by treating the original test as an integration rather than unit test & making sure that it passes with real (not mock) instances of the new classes until after you've completed the refactor.

All that said - I don't think this type of change is a prerequisite for merging the code. It's more important that the job can be configured (and by default is configured) as functionally identical to the original job, but can also be reconfigured to use the `ready_for_status_check` flag instead.

Reply:
@NavaTim Yeah, I'm now spending more time on this refactor than the actual requirements of the ticket, so I think I'm going to abandon them for the time being and open a separate ticket.